changelogs/unreleased/fix-inmanta-warnings.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: Fixes an issue about InmantaWarnings that does not exist anymore.
+change-type: patch
+destination-branches: [master]
```

Comment:
the branch for iso5 hasn't been merged yet (it failed on the same issue) so I've directly fixed it in this PR: https://github.com/inmanta/inmanta-core/pull/5049